### DISCUSSION TITLE
In TX_CONVERTIBLE_ISSUANCE, should conversion_discount and conversion_fixed_ownership be mutually exclusive?

### DISCUSSION THREAD

**Discussion by pjohnmeyer** [Open-Cap-Table-Coalition/Open-Cap-Format-OCF#85] [Suggestions] [answered]
Unclear if this should be a `oneof` relationship. Need to create issue for this.

_Originally posted by @tmccnnll in https://github.com/Open-Cap-Table-Coalition/Open-Cap-Format-OCF/pull/63#discussion_r789947885_

**Comment by pjohnmeyer** [comment]
@JSv4 do we want to add this to the Member Comment Period project?

**Comment by JSv4** [comment]
Strikes me that there are probably ways to have an instrument with both fields. Problem is, in that case, how to signal to the systems / users which value is operative and why.

**Reply by JSv4** [reply]
While I'm thinking about this, the [latest conversion spec proposal](https://github.com/Open-Cap-Table-Coalition/Open-Cap-Format-OCF/pull/175#issuecomment-1108047470) will solve this problem. You can create multiple conversion triggers per instrument, and you could clearly specify the trigger conditions for when you get a fixed percentage vs not. I don't think this needs to be considered by the standards working group anymore and can be solved with better data modeling.

**Comment by JSv4** [comment, marked as answer]
This is also a good topic for the Standards Working Group. Referring to that group.
